### DISCUSSION TITLE
7a96c83a-1d97-4904-a429-c71d5c7f3410

### DISCUSSION THREAD

**Discussion by giscus** [SSWConsulting/SSW.Rules.Content#9714] [Rules Discussions]
# 7a96c83a-1d97-4904-a429-c71d5c7f3410

Learn how .NET Aspire simplifies cloud-native development with orchestration, integrations, and tooling.

https://www.ssw.com.au/rules/aspire/

**Comment by TobyChurches** [comment]
I haven't use .NET Aspire before, but I look forward to when I get to! It sounds great!

**Comment by drwharris** [comment]
Really looking forward to getting my head over Aspire

**Comment by joshbermanssw** [comment]
Haven't yet used it but the rule and video from Wicksy sell it as a useful tool!

**Comment by danielmackay** [comment]
I find Aspire incredibly helpful when getting a new project up and running, and things are even better with .NET 9. Aspire is a no brainer for any project that has more than one moving part (e.g. API + Database). Besides the setup, the telemetry is super useful when tracing errors.

Aspire plays a massive role in providing a great DevEx which results in better development productivity and enjoyment on the project.

**Comment by miichaelsmedley** [comment]
Still learning and this seems to make the setting a project much simpler.

**Reply by awaltos** [reply]
Looking forward to using this in a project one day. Great vid by Matt. I have seen Will L demo this a couple times at user groups and it always looks easy to set up and use.

**Comment by isaaclombardssw** [comment]
Feels great to add some structure to a lot of moving parts.

**Comment by bahjatalaadelssw** [comment]
Anything that helps manage assets and keep a consistent execution.

**Comment by edkranz** [comment]
Looks cool, nice to see a clean UI for managing frontend and backend, and service discovery + all the other nice features seem like a big time saver – not only when working with .NET Aspire but also for onboarding new devs to a project with .NET Aspire.

**Comment by RobJT-wq** [comment]
Interesting rule, Excited to see what .Net Aspire can do for me

**Comment by steven0x51** [comment]
The setup instructions are clear and the process was buttery smooth. 
Very cool UI.

**Comment by Lewkans** [comment]
When you need to run multiple things at once, .NET Aspire makes the setup experience so much easier

**Comment by jakebayliss** [comment]
Looks awesome - very keen to learn and use Aspire!

**Comment by sam-s-maher** [comment]
I just became aware of Aspire recently, but already I can see how handy it can be - glad my current project uses it!

**Comment by Ben0189** [comment]
Aspire make developer's life easier

**Comment by jeoffreyfischer** [comment]
I am really keen to try Aspire! It seems to abstract so much of the usual complexity

**Reply by danielmackay** [reply]
You already tried it in the CA workshop recently. 😄

**Reply by jeoffreyfischer** [reply]
I was mostly copy/pasting though :p

**Comment by ncn-ssw** [comment]
I love how easy the dashboard is to follow and how easily the pieces of the project can be assembled. I really wish that I could use Aspire on my current project.

**Reply by stephanfakossw** [reply]
Looks awesome...but I'm not a developer

**Comment by MattParkerDev** [comment]
Aspire seems great for simplifying running complicated projects locally, however I'm not sure how much of "Aspire" translates to easy deployment to the cloud yet.

**Reply by danielmackay** [reply]
We'll cover how Aspire helps with deployments in the .NET Superpowers in January.

**Comment by sethdaily** [comment]
I aspire to wrap my head around this

**Comment by jimmidier** [comment]
Great video I'll definitely recommend .NET Aspire to the client on the next project

**Comment by amankumarrr** [comment]
I've heard of .NET Aspire before but never realized how useful it could be. Now, I'm looking forward to exploring and making the most of it!

**Comment by tino-liu** [comment]
I'm aspired to work on Aspire

**Comment by babakamyljanovssw** [comment]
Looking forward to try .NET Aspire in project, I think it can make development and monitoring efficient

**Comment by ZenoWang1999** [comment]
I take 3 min to set up the Aspire solution - Everything runs very quickly and smoothly. Since I'm not very clear about how Aspire works, I actually don't quite understand what happened. Anyway, it showed me a really cool interface.

**Comment by thom-wang** [comment]
Had great feelings about the dashboard, clean and neat and has so many informations

**Comment by Aibono1225** [comment]
Easy and fast setup. I'm keen to try Aspire!

**Comment by JackDevAU** [comment]
Keen to try it out! The local experience seems great! 
Will keep my eyes on this space 👀
Interested to see the deployment steps 👀👀

**Comment by Calinator444** [comment]
It's really great being able to visualize your app and endpoint running using the .Net Aspire dashboard

**Comment by ozair-ashfaque** [comment]
Thanks for the video Matt W. Looking forward to using this in a project.

**Comment by SamBWagner** [comment]
Aspire is pretty cool.

**Comment by vladislav-kir** [comment]
I haven't used Aspire before, but it looks cool

**Comment by bettybondoc** [comment]
Haven't used it before. Looks cool though!
